users/apps.py

```
@@ -1,5 +1,18 @@
 from django.apps import AppConfig
+import os
+from django.contrib.auth import get_user_model
+from django.db.utils import OperationalError, ProgrammingError
 
 class UsersConfig(AppConfig):
     default_auto_field = 'django.db.models.BigAutoField'
     name = 'users'
+
+    def ready(self):
+        deployment = os.environ.get('CIELO_DEPLOYMENT', '').lower()
```

Ruff (0.11.9)

Local variable `deployment` is assigned to but never used

Remove assignment to unused variable `deployment`

(F841)